test/test_files/tinysnb/function/string.test

```diff
@@ -144,6 +144,22 @@ False
 ---- 1
 Carol
 
+-NAME RegExprString
```

Comment:
RegExprString -> RegExprMatchString

Same for following.

Reply:
changed